src/inputs/EditableLabel/EditableLabel.js

```diff
@@ -50,7 +54,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export const EditableLabel = (props) => {
-  const { value, onNewValue, checkValue, labels, typographyProps, textFieldProps } = props;
+  const { value, onNewValue, checkValue, labels, typographyProps } = props;
```

Comment:
`textFieldProps` has not been removed at the bottom of the file, in `EditableLabel.propTypes`